apps/website/src/pages/docs/alert.mdx

```diff
@@ -2,7 +2,7 @@
 title: Alert
 description: A small box to quickly grab the user's attention and communicate a brief message.
 layout: ./_layout.astro
-propsPath: '@itwin/itwinui-react/src/core/Alert/Alert.tsx'
+propsPath: '@itwin/itwinui-react/esm/core/Alert/Alert.d.ts'
```

Comment:
Some of the other .mdx files have `/esm/` and some have `/src/`. Why is `/esm/` better? Because it's from the package vs straight from the code?